wurst/systems/commands/SpawningTestHelpers.wurst

```diff
@@ -75,6 +75,18 @@ let SPAWN_HELP_MESSAGE = compiletime(
     ).joinBy("\n")
 )
 
+let RESEARCH_HELP_MESSAGE = compiletime(
```

Comment:
All of this should go into the toolkit (current `lib/commands`), but I can move it there if you'd prefer. I also think you should check the type of the ID directly instead of what it starts with, but I can demonstrate that when I move it.

The reason I wanted you to make it generic is because I plan on moving that lib into its own dependency to separate out ITT-specific code versus generic libraries.

Reply:
I kind of see what you're planning, but I'll let you demonstrate if you don't mind.